.github/workflows/azure-ssh-attestation-build.yml

```diff
@@ -0,0 +1,34 @@
+name: azure-ssh-attestation-build
+
+on:
+  push:
+    branches: [ "quote-presentation" ]
+  pull_request:
+    branches: [ "quote-presentation" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+    - name: azure-demo ${{ github.sha }}
+      uses: appleboy/ssh-action@v1.0.3
+      with:
+         host: ${{ secrets.AZURE_TEE_BUILD_HOST }}
+         username: ${{ secrets.AZURE_BUILD_TEE_USERNAME }}
+         key: ${{ secrets.AZURE_BUILD_TEE_KEY }}
+         port: ${{ secrets.SSH_PORT }}
+         command_timeout: 120m
+         script: |
+            cd /tmp 
+            git clone https://github.com/tlsnotary/tlsn ${{ github.sha }}
+            cd ${{ github.sha }}
+            git checkout ${{ github.sha }}
+            cd crates/notary/server
+            : # if nc cant dial 7047 within 120m: ❌; else ✅ 
+            timeout 120 sh -c 'until nc -z $0 $1; do sleep 1; done' localhost 7047 &
+            /home/ubuntu/.cargo/bin/cargo run --bin notary-server --features tee_quote
```

Comment:
few questions:

1. Don't we need to kill the previous notary service that is running in the vm?
2. Since we don't have ACME yet, how is this notary server handling TLS?

Reply:
this is just a build script -- not to run it in prod, and is designed to support multiple concurrent builds
the tls issue will be addressed in the other pr and will be passed in as the config file so this build script should be oblivious